src/index.scss

```diff
@@ -61,3 +61,7 @@ body {
     background-color: $light-100;
   }
 }
+
+mark {
+  padding: 0;
+}
```

Comment:
This is for formatting the search results
Before:
![Screen Shot 2024-09-06 at 13 15 14](https://github.com/user-attachments/assets/bc45bd69-b5a4-403a-a507-b6895bf59210)
After:
![Screen Shot 2024-09-06 at 13 16 53](https://github.com/user-attachments/assets/5d715739-660d-48d5-a8e2-44b479d92b9f)